aiida_fleur/workflows/relax.py

```diff
@@ -340,12 +340,18 @@ def condition(self):
 
         largest_now = self.ctx.forces[-1]
 
+        # get force mixing (straight or BFGS) setting
+        # defaults to stright mixing if not set in scf.wf_parameters
+        if 'wf_parameters' in self.inputs.scf.wf_parameters:
+            force_strmix = self.inputs.scf.wf_parameters['force_dict']['forcemix'] == 'straight'
+        else:
+            force_strmix = False
```

Comment:
```suggestion
            force_strmix = True
```
I think the default in the scf workchain is straight mixing
See [here](https://github.com/JuDFTteam/aiida-fleur/blob/c8fbea6ba17eecabe5a367d5c6c16163b69c78d5/aiida_fleur/workflows/scf.py#L84)